sqlglot/dialects/duckdb.py

```diff
@@ -44,7 +44,12 @@
 from sqlglot.parser import binary_range_parser
 
 DATETIME_DELTA = t.Union[
-    exp.DateAdd, exp.TimeAdd, exp.DatetimeAdd, exp.TsOrDsAdd, exp.DateSub, exp.DatetimeSub
+    exp.DateAdd,
+    exp.TimeAdd,
+    exp.DatetimeAdd,
+    exp.TsOrDsAdd,
+    exp.DateSub,
+    exp.DatetimeSub,
```

Comment:
Did you do these formatting changes? Or did tooling get upgraded? I'd suggest reverting these to minimize visual noise for this PR.

Reply:
👍 i think that was my IDE